app/_hub/kong-inc/ai-semantic-cache/overview/_index.md

```diff
@@ -110,9 +110,9 @@ The plugin respects cache control headers to determine if requests and responses
 
 ## Get started with the AI Semantic Caching plugin
 
-* [Configuration reference](/hub/kong-inc/ai-semantic-caching/configuration/)
-* [Basic configuration example](/hub/kong-inc/ai-semantic-caching/how-to/basic-example/)
-* [Learn how to use the plugin](/hub/kong-inc/ai-semantic-caching/how-to/)
+* [Configuration reference](/hub/kong-inc/ai-semantic-cache/configuration/)
+* [Basic configuration example](/hub/kong-inc/ai-semantic-cache/how-to/basic-example/)
+* [Learn how to use the plugin](/hub/kong-inc/ai-semantic-cache/how-to/)
```

Comment:
```suggestion
* [Learn how to use the plugin](/hub/kong-inc/ai-semantic-cache/how-to/basic-example/)
```
I think that's the right link, the other gave me a 404 still

Reply:
Hmm. This is just a page that doesn't exist at all.